test/jdk/TEST.ROOT

```diff
@@ -9,17 +9,41 @@
 # should be taken to handle test failures of intermittent or
 # randomness tests.
 #
-# A "headful" test requires a graphical environment to meaningfully
-# run. Tests that are not headful are "headless".
-# A test flagged with key sound needs audio devices on the system, this
-# may be accompanied by the headful keyword since audio device access
+# A test flagged with cgroups uses cgroups.
+#
+# Notes on "client" keywords : headful sound printer 2d dnd multimon 
+# ==================================================================
```

Comment:
The notes reads well but they're notes for "2d" and "dnd". The heading hints that there will be information on these keywords.